data/warnings.json

```diff
@@ -0,0 +1,8 @@
OUBLE GLYPH", "KRASHOMNIS GLYPH", "KR1PTONPLAYER GLYPH", "KIWAD GLYPH", "KINGWOLBY GLYPH", "KINGKONGDONK GLYPH", "KINGGOTHALION GLYPH", "KAOIJI GLYPH", "KALON75 GLYPH", "KINGKACCHI GLYPH", "K0YI GLYPH", "JUSTPLAYBP GLYPH", "JOEYZERO GLYPH", "JAYNEVERWHERE GLYPH", "JANDONCOM GLYPH", "J3UBBLEBOY GLYPH", "IWOPLY GLYPH", "IVORYSMOON GLYPH", "INFRAREDMIKE GLYPH", "INFODIVERSAO GLYPH", "INEXPENSIVEGAMER GLYPH", "DOBRATV GLYPH", "HYNKARDSECTOR GLYPH", "HYDROXATE GLYPH", "HOTS'HOM STORIES GLYPH", "HOMI INVOCADO GLYPH", "FUJI GLYPH", "GRIND HARD SQUAD GLYPH", "GRIMLOCKE PRIME GLYPH", "GRIDDARK GLYPH", "GORILLAWOLFGAMING GLYPH", "GOOPERATIVES GLYPH", "GOKU707 GLYPH", "GOGUMAGAMING GLYPH", "GLITCHYGIRL GLYPH", "GLAM SHATTERSKULL GLYPH", "GIR8TACOS GLYPH", "GAMING B*TCHES GLYPH", "FROZENBAWZ GLYPH", "FROMTHE70S GLYPH", "FRANCOIS878 GLYPH", "FLUFFY MIRACLE GLYPH", "FATTSHANE GLYPH", "FATEFUL GLYPH", "ESP4HIM GLYPH", "ENDOTTI GLYPH", "EMOVJ GLYPH", "ELIGIBLEMONSTER GLYPH", "EL TIO PROD GLYPH", "JUNO GLYPH", "EDUIY16 GLYPH", "DRAMAKINS GLYPH", "DISTANT OBSERVER GLYPH", "DIMITRIV2 GLYPH", "DEUCETHEGAMER GLYPH", "DEN TW GLYPH", "DEEPBLUEBEARD GLYPH", "DEEJAYKNIGHT GLYPH", "TOTALN3WB GLYPH", "DAYJOBO GLYPH", "DARKFREACK GLYPH", "DARKSLAYER313 GLYPH", "DAPPADANMAN GLYPH", "DADEFUYE GLYPH", "DADXGOTXGAME GLYPH", "DNEXUS GLYPH", "DJTECHLIVE GLYPH", "D4NK3R GLYPH", "CRUSADER GLYPH", "CRESTBEAR GLYPH", "JAEMZ GLYPH", "WARFRAME COMMUNITY DISCORD GLYPH", "COHHCARNAGE GLYPH", "CRUXEDKAT GLYPH", "CHAR GLYPH", "CERATIA GLYPH", "CAPTAINTUTU GLYPH", "CAPTWALKER_1 GLYPH", "CANAL TENNO GLYPH", "CGS_KNACKIE GLYPH", "CASTIELIX GLYPH", "BWANA GLYPH", "BURNBXX GLYPH", "BUFF00N GLYPH", "BROZIME GLYPH", "BRICKYORCHID8 GLYPH", "BRICKY GLYPH", "BLACKONI GLYPH", "BIKEMAN GLYPH", "BIGJIMID GLYPH", "BENNYFITS GLYPH", "BAST 50 GLYPH", "BADNEWSBARON GLYPH", "BACKYARDISTV GLYPH", "BABYCHAN TH GLYPH", "BBSCHAINWARDEN GLYPH", "AZRAEL112 GLYPH", "AYWI GLYPH", "AURORASTARR GLYPH", "AUNGELECETTE GLYPH", "ASHI GLYPH", "ARGONSIX GLYPH", "ANNOYING KILLAH GLYPH", "ANGRYICEBERG GLYPH", "ANJETCAT GLYPH", "ALYEKK GLYPH", "ALTHANITHENOBLE GLYPH", "ALEXANDRALIVE GLYPH", "AJINGOM GLYPH", "AEONKNIGHT86 GLYPH", "ADMIRAL BAHROO GLYPH", "ADIKDARKCERO GLYPH", "ADELFOS SELENE GLYPH", "ACCESSIBLEGAMER GLYPH", "A H R GLYPH", "13ANGTV GLYPH", "ILLUMINA JUDICIUM", "WYRMIUS", "FRAME FIGHTER", "HAPPY ZEPHYR", "EXODIA BRAVE", "EXODIA VALOR", "PAX SOAR", "EXODIA MIGHT", "EXODIA HUNT", "PAX SEEKER", "EXODIA TRIUMPH", "EXODIA FORCE", "VIRTUOS TEMPO", "VIRTUOS SHADOW", "VIRTUOS STRIKE", "VIRTUOS FURY", "VIRTUOS NULL", "VIRTUOS GHOST", "VIRTUOS SURGE", "VIRTUOS FORGE", "VIRTUOS SPIKE", "VIRTUOS TROJAN", "MAGUS HUSK", "MAGUS FIREWALL", "MAGUS ELEVATE", "MAGUS NOURISH", "MAGUS REPLENISH", "MAGUS REPAIR", "MAGUS VIGOR", "MAGUS MELT", "MAGUS ACCELERANT", "MAGUS DRIVE", "MAGUS CLOUD", "MAGUS GLITCH", "MAGUS DESTRUCT", "MAGUS ANOMALY", "MAGUS REVERT", "MAGUS OVERLOAD", "MAGUS CADENCE", "MAGUS LOCKDOWN", "PAX CHARGE", "PAX BOLT" ]
```

Comment:
did we remove the types for the arcanes? i'm seeing a lot of them in here, as well as primary/secondary weapons

Reply:
I really like the idea of the file overall, though.

Reply:
Arcanes never worked, see #22 
Where are you seeing them though? I can't find them looking for `Arcane`

Reply:
I had a fix for those on a previous build that picked up most of them...

Reply:
Rip, i never got that in or merged. I'll take a look after this is all in.